packages/@uppy/core/src/Uppy.ts

```diff
@@ -29,7 +29,7 @@ import throttle from 'lodash/throttle.js'
 // @ts-ignore untyped
 import ee from 'namespace-emitter'
 import { nanoid } from 'nanoid/non-secure'
-import type { h } from 'preact'
+import { h } from 'preact'
```

Comment:
Seems like all these changes should happen elsewhere? Or did you pull main into your branch while 5.0 is outdated?